### DISCUSSION TITLE
UDP packet loss investigation

### DISCUSSION THREAD

**Discussion by mattmelton** [SteamRE/SteamKit#1250] [General]
I'm investigating UDP packet loss between London and Amsterdam when I play Dota 2. I want to parse [the UDP data header sequence number](https://github.com/ValveSoftware/GameNetworkingSockets/blob/master/src/steamnetworkingsockets/clientlib/steamnetworkingsockets_udp.cpp#L671) so I can write a UDP echo server that can I can detect when packet loss occurs. Does this project have a protobuf definition or generated class that can help?

**Comment by zxcvqwerasdf** [comment]
Does this project have a protobuf definition or generated class that can help? - Yes.
But encryption/compression and establishment connection to the game server is not described here.
